### PR TITLE
Simplify Browserstack test runner code

### PR DESCRIPTION
**Pull request by Turbo87**
/cc @kategengler @rwjblue 

bildschirmfoto 2018-12-06 um 01 26 48

### CODE REVIEW

File: bin/run-browserstack-tests.js [hunk 1]
```diff
@@ -1,71 +1,39 @@
-/* eslint-disable no-console */
+/* eslint-disable no-console, node/no-unsupported-features */
 
-var RSVP = require('rsvp');
-var spawn = require('child_process').spawn;
+var execa = require('execa');
```
Comment (rwjblue) [inline bin/run-browserstack-tests.js]: Seems _generally_ odd to me to use `async` / `await` **and** `var` in the same file... 😹

File: bin/run-browserstack-tests.js [hunk 2]
```diff
@@ -1,71 +1,39 @@
...
-  .finally(function() {
-    var promise = RSVP.resolve();
-    if (process.env.TRAVIS_JOB_NUMBER) {
-      promise = run('./node_modules/.bin/ember', ['browserstack:results']);
+(async function() {
+  await run('ember', ['browserstack:connect']);
+
+  try {
+    try {
+      // Calling testem directly here instead of `ember test` so that
```
Comment (rwjblue) [inline bin/run-browserstack-tests.js]: Hmm, we could avoid the double build **and** still use `ember test` by way of `ember test --path=dist`, any objection?

Comment (Turbo87) [reply bin/run-browserstack-tests.js]: I'm not sure what the advantage would be over just calling `testem` directly 🤔

File: bin/run-browserstack-tests.js (hunk 1 above)
Comment (Turbo87) [reply bin/run-browserstack-tests.js]: agreed, changed to `const`

**Pull request merged**